janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.diskstorage.cassandra.thrift;
+
+import org.janusgraph.CassandraStorageSetup;
+import org.janusgraph.diskstorage.configuration.ModifiableConfiguration;
+import org.junit.Test;
+
+public class CassandraThriftStoreManagerTest {
+
+    @Test(expected = IllegalArgumentException.class)
+    public void configOptionFrameSizeMbShouldErrorOnLowValue() {
+        ModifiableConfiguration config = CassandraStorageSetup.getCassandraThriftConfiguration("janusgraph");
+        config.set(CassandraThriftStoreManager.THRIFT_FRAME_SIZE_MB, 0);
+        config.get(CassandraThriftStoreManager.THRIFT_FRAME_SIZE_MB);
+    }
+
+    @Test
+    public void configOptionFrameSizeMbShouldBeHappy() {
+        ModifiableConfiguration config = CassandraStorageSetup.getCassandraThriftConfiguration("janusgraph");
+        config.set(CassandraThriftStoreManager.THRIFT_FRAME_SIZE_MB, 1);
+        config.get(CassandraThriftStoreManager.THRIFT_FRAME_SIZE_MB);
```

Comment:
Should this verify that the returned value is 1, not just that it's not crashing?

Reply:
@mbrukman Sorry, it looked like your earlier feedback was addressed and I merged this too quick.

Reply:
No worries, it's all good. The beauty of software is that it's always easier to change it (and then change it again) compared with hardware. :-)

Reply:
This was fixed by @robertdale in PR #560.